shared/common-adapters/list-item2.tsx

```diff
@@ -193,9 +199,6 @@ const styles = Styles.styleSheetCreate({
     right: 0,
     top: 0,
   },
-  hiddenNoHoverAction: {
-    justifyContent: 'flex-end',
```

Comment:
@jakob223 This doesn't seem used as the actions are passed in as a box. Let me know if I'm wrong!

Reply:
This is used - it prevents the actions from moving as they are uncovered in the 'grow' case, and instead keeps them in one place as the avatars move out from on top of them.